test/extensions/filters/network/common/fuzz/network_readfilter_fuzz_test.cc

```diff
@@ -9,41 +9,84 @@
 #include "test/fuzz/fuzz_runner.h"
 #include "test/test_common/test_runtime.h"
 
+#include "libprotobuf_mutator/src/libfuzzer/libfuzzer_macro.h"
+#include "src/text_format.h" // from libprotobuf_mutator/
```

Comment:
Huh, how does the compiler know that "src" is in libprotobuf_mutator? doesn't it need the same prefix as the line above?

FWIW, this seems to be coupling with the internals of the mutator library, which isn't a desired state IMHO. IIUC we use these for the serialization/deserialization to/from protobuf. Envoy has its own methods that do something similar, so we may want to consider using them. assuming I didn't miss anything, please add a TODO above this line that mentions which functions need to be replaced.

Reply:
Yes, that is ridiculous. But the `src/textformat.h` is referenced from the source directory while the fuzzer_macros in the line above are taken from the build directory. Therefore these two different paths. It could also be, that I am doing something wrong in the `BUILD` recipe, but I did not find another way to get this working.

I added the TODO. Didn't find the appropriate functions ad hoc.